scripts/main.js

```diff
@@ -51,8 +53,18 @@ $goBackButton.on('click', function () {
   setRoute('')
 })
 
+$editButton.on('click', function(event){
+  console.log('test')
+$body.attr('data-state', 'update-app')
+//  if($submitButton.clicked == true)
+// {
+//
+// }
+})
 $updateAppForm.on('submit', function (event) {
   event.preventDefault()
+  //rendeupdateform()
+
   var changed = $('#rename-app').val()
   var app = {
     id: $('#name-app').data('id'),
```

Comment:
just as a reminder, the problem was with this line. `$('#name-app').data('id')` returned an empty string instead of the app id